dev_guide/ssh_environment.adoc

```diff
@@ -0,0 +1,37 @@
+= Managing Applications in a Secure Shell Environment
```

Comment:
This is singular "an SSH env" while the `_build_cfg.yml` Name is plural.  Suggest changing one to be consistent w/ the other.